sierralocal/nucaminohook.py

```diff
@@ -278,12 +278,19 @@ def align_file(self, filename, program='post'):
                                 if not protein['Error'] and protein['Report'] and 'pol' in protein['Gene']:
                                     for key, info in protein['Report'].items():
                                         if key == 'AlignedSites':
-                                            for i in info:
-                                                # PosNA shows all three positions, should only show first
-                                                i['PosNA'] = i['PosNAs'][0]
-                                                i.pop('PosNAs')
-                                                i['PosAA'] = i['PosAA'] + 1
-
+                                            i = 0
+                                            while i < len(info):
+                                                # len of 0 breaks code, need to check if len of 1 or 2 breaks it too
+                                                if info[i]['LengthNA'] == 0:
+                                                    del info[i]
```

Comment:
This seems unsafe to me - can't we just splice out this position from the list, *i.e.*, `info = info[:i] + info[(i+1):]`